performance/sqlserver/memory_settings.rst

```diff
@@ -0,0 +1,21 @@
+Memory Settings
+===============
+
+The total ammoun of memory needed by SQL Server depends on your use case and (expected) workload. SQL Server will claim memory from the OS when needed but will not give it back to the OS once claimed. When not specifying a limit you will run the risk of starving the host server and OS of enough free memory to operate efficently. It is therefore strongly advised to limit the maximum ammount SQL Server will use.
```

Comment:
- Typo: 'ammoun' -> 'amount'
- Style: 'When not specifying a limit' -> 'If no limit is specified'
- Typo: 'efficently' -> 'efficiently'
- Typo: 'ammount' -> 'amount'
- Style: 'will use' -> 'is allowed to use'

Reply:
Corrected typo's